packages/js/e2e-utils/src/flows/with-rest-api.js

```diff
@@ -138,7 +138,7 @@ export const withRestApi = {
 		if ( productCategories.data && productCategories.data.length ) {
 			for ( let c = 0; c < productCategories.data.length; c++ ) {
 				// The default `uncategorized` category can't be deleted
-				if ( productCategories.data[c].id == 0 ) {
+				if ( productCategories.data[c].slug == 'uncategorized' ) {
```

Comment:
This should have a `Fixed` changelog entry.

Reply:
Thanks for catching this. changelog updated.